src/hooks/protections/receive_net_message.cpp

```diff
@@ -259,7 +259,7 @@ namespace big
 		if (g.debug.logs.packet_logs) [[unlikely]]
 		{
 			if (g.debug.logs.packet_logs == 1 || //ALL
-			   (g.debug.logs.packet_logs == 2 && msgType != rage::eNetMessage::MsgCloneSync && msgType != rage::eNetMessage::MsgPackedCloneSyncACKs && msgType != rage::eNetMessage::MsgPackedEvents && msgType != rage::eNetMessage::MsgPackedReliables && msgType != rage::eNetMessage::MsgPackedEventReliablesMsgs && msgType != rage::eNetMessage::MsgNetArrayMgrUpdate && msgType != rage::eNetMessage::MsgNetArrayMgrSplitUpdateAck && msgType != rage::eNetMessage::MsgNetArrayMgrUpdateAck && msgType != rage::eNetMessage::MsgScriptHandshakeAck && msgType != rage::eNetMessage::MsgScriptHandshake && msgType != rage::eNetMessage::MsgScriptJoin && msgType != rage::eNetMessage::MsgScriptJoinAck && msgType != rage::eNetMessage::MsgScriptJoinHostAck && msgType != rage::eNetMessage::MsgRequestObjectIds && msgType != rage::eNetMessage::MsgInformObjectIds && msgType != rage::eNetMessage::MsgNetTimeSync)) //FILTERED
+			    (g.debug.logs.packet_logs == 2 && msgType != rage::eNetMessage::MsgCloneSync && msgType != rage::eNetMessage::MsgPackedCloneSyncACKs && msgType != rage::eNetMessage::MsgPackedEvents && msgType != rage::eNetMessage::MsgPackedReliables && msgType != rage::eNetMessage::MsgPackedEventReliablesMsgs && msgType != rage::eNetMessage::MsgNetArrayMgrUpdate && msgType != rage::eNetMessage::MsgNetArrayMgrSplitUpdateAck && msgType != rage::eNetMessage::MsgNetArrayMgrUpdateAck && msgType != rage::eNetMessage::MsgScriptHandshakeAck && msgType != rage::eNetMessage::MsgScriptHandshake && msgType != rage::eNetMessage::MsgScriptJoin && msgType != rage::eNetMessage::MsgScriptJoinAck && msgType != rage::eNetMessage::MsgScriptJoinHostAck && msgType != rage::eNetMessage::MsgRequestObjectIds && msgType != rage::eNetMessage::MsgInformObjectIds && msgType != rage::eNetMessage::MsgNetTimeSync)) //FILTERED
```

Comment:
Unncessary whitespace change as usual.

Reply:
Per usual, I run the formatter

Reply:
Have you considered not doing that? You know, like everyone else?

Reply:
No, I run the formatter via VS code and it formats the whole file. If it were up to date/formatted in the first place there would be no problems. Also, I did not personally make the change, in fact the formatter did and it follows the instructions that are predefined made the change, so consider editing that if you have a big problem with it.

Reply:
i think the best practice should be only to format piece of code you work on and not whole file given the existing file which was created unformatted. New files should always be formatted from top to bottom.

Reply:
> > i think the best practice should be only to format piece of code you work on and not whole file given the existing file which was created unformatted. New files should always be formatted from top to bottom.
> 
> 
> 
> This is the way literally everyone else including @Yimura does it. 

Yimura said this is permittable, so for the final time I am going to mark it as resolved because it is.

Reply:
there are two major problems with formatting from my personal experience and god knows if forget something more -
1. you add commit changes for something you only formatted and made no actual contribution to it. When time-travelling back in history to see who added the actual code it becomes mess sometimes?
2. there are sometimes complicated formatted changes which can cause issues related to merge when multiple people work on a given branch i don't remember perfectly there was some issue related to this.